testapps/php56_e2e/web/grpc_pubsub.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+require_once 'empty.php';
+require_once 'pubsub.php';
+require_once 'timestamp.php';
+require_once 'descriptor.php';
+require_once 'http.php';
+require_once 'annotations.php';
```

Comment:
We could add these to the autoloader with composer's [classmap directive](https://getcomposer.org/doc/04-schema.md#classmap) in [`testapps/php56_e2e/composer.json`](https://github.com/GoogleCloudPlatform/php-docker/blob/master/testapps/php56_e2e/composer.json):

```php
{
    ...
    "autoload": {
        "classmap": ["web"]
    }
}
```